portfolio.css

```diff
@@ -29,10 +32,81 @@ body {
     transform: translate(-50%, -50%);
 }
 
-.page-links-container-portfolio {
-    /* Vertically and horizontally centre div */
-    position: absolute;
-    top: 95%;
-    left: 50%;
-    transform: translate(-50%, -50%);
-}
\ No newline at end of file
+.icons {
+    margin-top: 70px;
+    text-align: center; /* How to center inline elements*/
+}
+
+/*.page-links-container-portfolio {*/
```

Comment:
Remember to remove comments or reactivate code when pulling into master.

Reply:
will do